notebooks/00_quick_start/sar_aml_compute_movielens.ipynb

```diff
@@ -0,0 +1,477 @@
+{
```

Comment:
<p>there is an ongoing conversation to discuss what is the best place to locate this notebook, it is on teams right now</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/575/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Heather left a comment in this PR about this discussion. We decided not to have a separate aml folder.